frontend/src/metabase/sharing/components/SharingSidebar.jsx

```diff
@@ -709,6 +709,7 @@ class SharingSidebar extends React.Component {
           onClose={this.handleSave}
           onCancel={onCancel}
           className="text-dark"
+          closeIsDisabled={!pulseIsValid(pulse, formInput.channels)}
```

Comment:
@tsmacdonald Mind giving a little color on how this prop and the `Done` button relate?

Reply:
The `Done` button auto-magically appears if `onClose` is set, and it will be disabled if `closeIsDisabled` is true. 

I totally just cargo-culted using it from the normal Pulse UI.

Reply:
Cool. That prop name is weird but I pulled the branch and everything seems to be working as expected. Might just add a quick code comment to help whoever comes along next decipher that a bit faster.